docs/book/installation/vagrant_installation.rst

```diff
@@ -0,0 +1,57 @@
+.. index::
+   single: Installation via Vagrant
+
+Sylius installation via Vagrant
+===============================
+
+.. warning::
+
+    This article assumes you're familiar with `Composer`_, a dependency manager
```

Comment:
This article assumes you're familiar with `Composer`, `Vagrant` and `VirtualBox`